.github/workflows/vast.yaml

```diff
@@ -513,3 +522,94 @@ jobs:
           # for builds of the latest release.
           asset_name: "${{ steps.configure_env.outputs.publish_name }}.tar.gz"
           asset_content_type: application/gzip
+
+  plugins:
+    runs-on: ubuntu-20.04
+    # run everything inside a Debian container on an ubuntu-20.04 host
```

Comment:
this comment should be moved down by one line.

Reply:
I opted to remove it, it's a pretty meaningless comment.